src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -161,8 +170,14 @@ class AddressValidationModal extends React.Component {
       validationKey,
       addressValidationError,
       closeModal,
+      resetAddressValidation,
     } = this.props;
 
+    const resetDataAndCloseModal = () => {
```

Comment:
Rather than simply closing the Address Validation Modal we should clean up after ourselves and remove the address validation data from the redux store. This change isn't required to make the user-entered address show up when they go back to the edit form but it felt like a good thing to add.